python/fledge/plugins/south/csvplayback/csvplayback.py

```diff
@@ -101,15 +173,16 @@
         'validity': "ingestMode == \"burst\"",
         'displayName': 'Burst interval (ms)',
         'minimum': '1',
-        'order': '6'
+        'order': '14'
     },
     'timestampStyle': {
         'description': 'Select "continuous" mode asset timestamp processing style.',
         'type': 'enumeration',
         'default': 'current time',
+        'validity': "variableCols == \"false\"",
         'options': ['current time', 'copy csv value', 'move csv value', 'use csv sample delta'],
```

Comment:
Keep options value the same way for others i.e no snake_case or camelCase. Here make it Current time, Copy..., Move..., Use...